propagator/opentelemetry-propagator-aws-xray/src/opentelemetry/propagators/aws/aws_xray_propagator.py

```diff
@@ -328,9 +328,9 @@ def fields(self):
         return {TRACE_HEADER_KEY}
 
 
-class AwsXrayLambdaPropagator(AwsXRayPropagator):
+class AwsXRayLambdaPropagator(AwsXRayPropagator):
```

Comment:
We should probably add a test that asserts that the entry points are loadable

Reply:
Right, fixed

Reply:
I meant for the propagator entry point

Reply:
Added a test